Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
For example, maybe ucal_add doesn't do what we want when told to add one WeekOfMonth to a date? If that's the case, maybe calculate should set fieldToAdd to Week instead of WeekOfMonth. Just speculating.

---
In reply to: [74354002](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74354002) [](ancestors = 74354002,74342094,74336418)